lib/presentation/features/heroes/bloc/heroes_bloc.dart

```diff
@@ -3,33 +3,35 @@ import 'package:marvel/domain/entities/character.dart';
 import 'package:marvel/domain/use_cases/get_characters_use_case.dart';
 import 'package:marvel/presentation/features/heroes/bloc/heroes_event.dart';
 import 'package:marvel/presentation/features/heroes/bloc/heroes_state.dart';
-import 'package:marvel/presentation/convertors/convertor.dart';
-import 'package:marvel/presentation/view_models/view_data_character.dart';
+import 'package:marvel/presentation/converters/converter.dart';
+import 'package:marvel/presentation/view_data/view_data_character.dart';
 
 class HeroesBloc extends Bloc<HeroesEvent, HeroesState> {
   final GetCharactersUseCase getCharactersUseCase;
 
   HeroesBloc({required this.getCharactersUseCase})
       : super(
-          const HeroesState(loading: false, hasReachedMax: false),
+          const HeroesState(loading: false, hasReachedMax: false, error: false),
         ) {
     on<ReadyForDataEvent>(
       (event, emit) async {
         emit(
           state.copyWith(loading: true),
         );
         try {
-          final List<Character> characters = await getCharactersUseCase.call();
-          final List<ViewDataCharacter> charactersViewData = _mapCharacters(characters);
+          final List<Character> localCharacters = await getCharactersUseCase.call();
```

Comment:
```suggestion
          final List<Character> localCharacters = await getCharactersUseCase;
```
`call()` is mostly used with nullable objects to have a chance to have an optional call like `nullableObject?.call()` but in case is non-nuulable it can be used like this `nonNullableObject()`. 
It is just small syntetig sugar - your varian with call is also fine.